strings/tcg/set_one.json

```diff
@@ -1,1506 +1,1506 @@
 		{
-			"id": "bepis",
-			"name": "B.E.P.I.S. Chamber",
+			"cardsubtype": "Instant",
+			"cardtype": "Event",
 			"desc": "Created as an automated investment machine for a venture capitalism company, the B.E.P.I.S. ended up in the hands of Nanotrasen's research division after bankrupting the original creators... and 27 other corporations.",
-			"rules": "Flip a coin. If heads, gain 2 mana. If tails, lose up to 2 mana.",
+			"faction": "Science",
 			"icon_state": "bepis",
+			"id": "bepis",
+			"name": "B.E.P.I.S. Chamber",
 			"power": "0",
+			"rarity": "common",
 			"resolve": "0",
-			"faction": "Science",
-			"summoncost": "0",
-			"cardtype": "Event",
-			"cardsubtype": "Instant",
+			"rules": "Flip a coin. If heads, gain 2 mana. If tails, lose up to 2 mana.",
 			"series": "coreset2020",
-			"rarity": "common"
+			"summoncost": "0"
 		},
 		{
-			"id": "weekend_at_bernies",
-			"name": "Weekend at Nanotrasen's",
+			"cardsubtype": "Instant",
+			"cardtype": "Event",
 			"desc": "Holy shit! We just accidentally killed the Captain! Here, uhh... oh god, stuff him in that wheelchair over there, before anyone comes in. Yeah, yeah! Get his sunglasses, cover his face. Oh my god, oh my god, oh my god. No one will be able to tell, he looks sorta fine, right? Right?",
-			"rules": "Return a creature card from your graveyard to the battlefield.",
+			"faction": "Medical",
 			"icon_state": "weekend_at_bernies",
+			"id": "weekend_at_bernies",
+			"name": "Weekend at Nanotrasen's",
 			"power": "0",
+			"rarity": "uncommon",
 			"resolve": "0",
-			"faction": "Medical",
-			"summoncost": "3",
-			"cardtype": "Event",
-			"cardsubtype": "Instant",
+			"rules": "Return a creature card from your graveyard to the battlefield.",
 			"series": "coreset2020",
-			"rarity": "uncommon"
+			"summoncost": "3"
 		},
 		{
-			"id": "disco_inferno",
-			"name": "Disco Inferno",
+			"cardsubtype": "Shuttle",
+			"cardtype": "Battlefield",
 			"desc": "(Burn baby burn) burn that mother down y'all\n(Burn baby burn) Disco Inferno\n(Burn baby burn) burn that mother down",
-			"rules": "For 2 mana: Tap this card permanantly. While tapped, all active creatures take 3 damage during the first play phase of each turn. This card is destroyed after 2 turns of being tapped.",
+			"faction": "Science",
 			"icon_state": "disco_inferno",
+			"id": "disco_inferno",
+			"name": "Disco Inferno",
 			"power": "0",
+			"rarity": "uncommon",
 			"resolve": "0",
-			"faction": "Science",
-			"summoncost": "4",
-			"cardtype": "Battlefield",
-			"cardsubtype": "Shuttle",
+			"rules": "For 2 mana: Tap this card permanantly. While tapped, all active creatures take 3 damage during the first play phase of each turn. This card is destroyed after 2 turns of being tapped.",
+			"series": "coreset2020",
+			"summoncost": "4"
+		}
+	],
+	"templates": [
+		{
+			"icon": "icons/runtime/tcg/default.dmi",
 			"series": "coreset2020",
-			"rarity": "uncommon"
+			"template": "default"
 		}
 	]
 }
```

Comment:
This should not be at the bottom
In general the card files are setup in a way that semi defies sorting, might be best to just let them alone

Reply:
agreed, will reverse